docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -350,6 +350,23 @@ The following table includes some additional commands that might help you manage
 | Runs one iteration of `logrotate` manually, without changing the scheduled interval. | `sudo logrotate -f /etc/logrotate.conf --force` |
 | Reloads `syslog-ng`. | `sudo /usr/sbin/syslog-ng-ctl reload` |
 
+### UID2 Operator error codes
```

Comment:
Initial cap for headings: also, per discussion, it should be one level up:

## UID2 Operator Error Codes